electrochemicalquantities.ttl

```diff
@@ -780,8 +793,9 @@ and t* through mathematical models, provided that the long-time potential- deter
 ###  http://emmo.info/electrochemistry#electrochemistry_457aed58_6e7e_42d5_9d78_b424c8d60aa3
 :electrochemistry_457aed58_6e7e_42d5_9d78_b424c8d60aa3 rdf:type owl:Class ;
                                                        rdfs:subClassOf emmo:EMMO_faab3f84_e475_4a46_af9c_7d249f0b9aef ;
-                                                       skos:altLabel "DC-Per"@en ,
-                                                                     "the percentage of the total discharge capacity that is obtained during a constant current discharge process"@en ;
+                                                       skos:altLabel "the percentage of the total discharge capacity that is obtained during a constant current discharge process"@en ;
```

Comment:
I believe this should be a comment or elucidation, not an altLabel

Reply:
thanks. done.